subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -504,7 +504,7 @@ uint8_t ll_cis_create_check(uint16_t cis_handle, uint16_t acl_handle)
 
 		/* Verify handle validity and association */
 		cis = ll_conn_iso_stream_get(cis_handle);
-		if (cis->lll.handle == cis_handle && cis->lll.acl_handle == acl_handle) {
+		if (cis->lll.handle == cis_handle) {
```

Comment:
Just to make sure: The `handle`/`cis_handle` here are the connection handle, and not the CIS ID, right?

Reply:
these are connection handles, not CIS ID.